habs_portal/static/js/partials/StationMetadata.html

```diff
@@ -0,0 +1,14 @@
+<% collection.each(function(model) { %>
```

Comment:
@lukecampbell I'm not sure if this proper backbone etiquette, but I'm sending a collection to the metadata panel even though this collection will only have most 1 record (the active station).